diff --git a/node_modules/life_star/.envrc b/node_modules/life_star/.envrc
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/.travis.yml b/node_modules/life_star/.travis.yml
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/Jakefile b/node_modules/life_star/Jakefile
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/README.md b/node_modules/life_star/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/flake.lock b/node_modules/life_star/flake.lock
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/flake.nix b/node_modules/life_star/flake.nix
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/lib/auth.js b/node_modules/life_star/lib/auth.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/lib/manifest.js b/node_modules/life_star/lib/manifest.js
old mode 100644
new mode 100755
index 2a9cebe..572179c
--- a/node_modules/life_star/lib/manifest.js
+++ b/node_modules/life_star/lib/manifest.js
@@ -173,7 +173,7 @@ ManifestHandler.prototype.addManifestRef = function(req, res) {
             + string.substring(idx + 6);
 	var contentLength = Number(interceptedHeaders['content-length']);
 	interceptedHeaders['content-length'] = contentLength + manifestRef.length;
-	buffers[i] = new Buffer(string);
+	buffers[i] = Buffer.from(string);
 	return buffers;
     }
     return buffers;
diff --git a/node_modules/life_star/lib/proxy.js b/node_modules/life_star/lib/proxy.js
old mode 100644
new mode 100755
index b0b08aa..2e63d0b
--- a/node_modules/life_star/lib/proxy.js
+++ b/node_modules/life_star/lib/proxy.js
@@ -50,7 +50,7 @@ module.exports = function(logger) {
   };
 
   var collectBody = function(req, next) {
-    var buffer = new Buffer(parseInt(req.header('Content-length')), 'binary').fill(0);
+    var buffer = Buffer.alloc(parseInt(req.header('Content-length')));
     var offset = 0;
     req.on('data', function(chunk) {
       chunk.copy(buffer, offset);
diff --git a/node_modules/life_star/lib/subservers.js b/node_modules/life_star/lib/subservers.js
old mode 100644
new mode 100755
index ebcfec9..e0c4f24
--- a/node_modules/life_star/lib/subservers.js
+++ b/node_modules/life_star/lib/subservers.js
@@ -255,7 +255,7 @@ SubserverHandler.prototype.registerWith = function(app, server) {
   app.post(metaURL + '/:name/unload', this.unload.bind(this));
   app.get(metaURL + '/:name', this.getSubserverSource.bind(this));
   app.put(metaURL + '/:name', this.setSubserverSource.bind(this));
-  app.del(metaURL + '/:name', this.deleteSubserver.bind(this));
+  app.delete(metaURL + '/:name', this.deleteSubserver.bind(this));
   app.get(metaURL, this.listSubservers.bind(this));
 
   // load and start the subservers
diff --git a/node_modules/life_star/lib/testing.js b/node_modules/life_star/lib/testing.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/lib/util.js b/node_modules/life_star/lib/util.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/life_star.js b/node_modules/life_star/life_star.js
old mode 100644
new mode 100755
index c03d3c3..972c0ee
--- a/node_modules/life_star/life_star.js
+++ b/node_modules/life_star/life_star.js
@@ -10,7 +10,6 @@ var lang             = require('lively.lang'),
     SubserverHandler = require('./lib/subservers').SubserverHandler,
     ManifestHandler  = require('./lib/manifest').ManifestHandler,
     lfUtil           = require('./lib/util'),
-    util             = require('util'),
     fs               = require('fs'),
     path             = require('path'),
     EventEmitter     = require('events').EventEmitter
@@ -72,7 +71,7 @@ var serverSetup = module.exports = function(config, thenDo) {
   function extendServerSetupFunction(next) {
     // make it an event emitter so that life_star users can receive server
     // related events. Events are emitted in startHTTPListener()
-    util._extend(serverSetup, EventEmitter.prototype);
+    Object.assign(serverSetup, EventEmitter.prototype);
     EventEmitter.call(serverSetup);
     
     serverSetup.getServer = function() { return server; };
@@ -84,7 +83,7 @@ var serverSetup = module.exports = function(config, thenDo) {
     // some helpers, mainly for interactive usage
     if (typeof global.lively === "undefined") global.lively = {};
     var lv = global.lv = global.lively;
-    util._extend(lv, {
+    Object.assign(lv, {
       server: {
         dir: __dirname,
         get lifeStar() { return server; },
@@ -380,7 +379,7 @@ var serverSetup = module.exports = function(config, thenDo) {
     if (config.dbConf) {
         if (typeof config.dbConf === 'string')
             config.dbConf = JSON.parse(config.dbConf);
-        util._extend(dbConf, config.dbConf);
+        Object.assign(dbConf, config.dbConf);
     }
     var fsHandler = new LivelyFsHandler(dbConf).registerWith(app, server);
     global.lively.server.repository = fsHandler.repository;
diff --git a/node_modules/life_star/standalone_star.js b/node_modules/life_star/standalone_star.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/subservers/.gitkeep b/node_modules/life_star/subservers/.gitkeep
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/tests/life_star-test-support.js b/node_modules/life_star/tests/life_star-test-support.js
old mode 100644
new mode 100755
index 90cb831..aa18305
--- a/node_modules/life_star/tests/life_star-test-support.js
+++ b/node_modules/life_star/tests/life_star-test-support.js
@@ -6,12 +6,11 @@ var lifeStar = require("./../life_star"),
     async = require('async'),
     path = require('path'),
     http = require('http'),
-    util = require('util'),
     server;
 
 function withLifeStarDo(test, func, options) {
   if (server) throw new Error('life_star already running!');
-  options = util._extend(options || {}, {dbConf: {enableVersioning: false, enableRewriting: false}});
+  options = Object.assign(options || {}, {dbConf: {enableVersioning: false, enableRewriting: false}});
   options.host = options.host || 'localhost';
   options.port = options.port || 9999;
   server = lifeStar(options, function(err, server) { func(server); });
diff --git a/node_modules/life_star/tests/manifest-test.js b/node_modules/life_star/tests/manifest-test.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/tests/subservers-test.js b/node_modules/life_star/tests/subservers-test.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/tests/test-helper.js b/node_modules/life_star/tests/test-helper.js
old mode 100644
new mode 100755
diff --git a/node_modules/life_star/tmp/dummy.txt b/node_modules/life_star/tmp/dummy.txt
old mode 100644
new mode 100755
